lib/git_reflow/git_server/git_hub/pull_request.rb

```diff
@@ -78,6 +78,45 @@ def approved?
           end
         end
 
+        def merge_feature_branch(feature_branch_name, options = {})
```

Comment:
@codenamev correct me if I am wrong, but it seems we still need to pass the `sha` in from `options`. I am not sure we can access `sha` from `source_object`